docker/monitoring/docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: "3.7"
+
+services:
+  zipkin:
```

Comment:
What do you think about defining the storage type, could be persisted to Cassandra (could be useful if Zipkin is shut down for any reason)

Reply:
I see no reason to persist traces.

Reply:
I don't have strong feelings about this either way. So I'm ok with it :)